test/fixtures/setup.js

```diff
@@ -56,4 +54,36 @@ module.exports = test => {
 				}
 			});
 	});
+
+	test.before(() => {
```

Comment:
Just put the code inside the other `test.before` so you don't have 2 `test.before` blocks.